OoyalaSkinSampleApp/app/build.gradle

```diff
@@ -87,10 +88,13 @@ dependencies {
     implementation files('libs/OoyalaSDK.aar')
     implementation files('libs/OoyalaSkinSDK.aar')
     implementation files('libs/OoyalaSSAISDK.aar')
+    implementation files('libs/OoyalaVRSDK.jar')
```

Comment:
let's not add VR to the OoyalaSkinSampleApp, we don't want to add everything in this one. Also removed all the files related to VR in this app.

Instead add an activity in the VR sample app to show how to use rate on that one.

Thanks.

Reply:
Ok. np. I'll do it tomorrow.